src/components/SellerProducts.js

```diff
@@ -37,22 +38,19 @@ export default class SellerProducts extends Component {
                           <p>{product.title}</p>
                         
                       </a>
-                      <span>
```

Comment:
Please pull the latest code. This change exists in repo but your commit shows this doesn't exist in your codebase.